src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java

```diff
@@ -79,11 +82,13 @@ protected byte[] createPdf(CaseData caseData) throws IOException {
         }
     }
 
-    private static String createPdfDocumentNameFromCaseData(CaseData caseData) {
+    private static String createPdfDocumentNameFromCaseData(CaseData caseData, String documentLanguage) {
         return "ET1_CASE_DOCUMENT_"
             + caseData.getClaimantIndType().getClaimantFirstNames().replace(" ", "_")
             + "_"
             + caseData.getClaimantIndType().getClaimantLastName().replace(" ", "_")
+            + "_"
+            + documentLanguage
```

Comment:
Please check if this is a breaking change to the frontend. There is a story coming to make sure the frontend picks the correct file from the document collection so its something to be mindful of https://tools.hmcts.net/jira/browse/RET-2782